From 055130476b2ae349ab00b5cc5958d02da2fc2983 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Date: Fri, 16 May 2025 16:12:03 +0200
Subject: [PATCH] CMakeLists.txt: add cmake_minimum_required()

Using cmake_minimum_required() is now mandatory with CMake 4.0.

Upstream: N/A, no clear upstream
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,3 +1,4 @@
+cmake_minimum_required(VERSION 3.18)
 PROJECT (cdrkit C)
 SUBDIRS(include genisoimage wodim libedc libhfs_iso libparanoia icedax libusal librols libunls readom netscsid 3rd-party/dirsplit)
 
